README.md

```diff
@@ -68,14 +74,16 @@ defmodule MyProject.Repo.Migrations.CreateMessage do
 end
 ```
 
-### Database model
+### Schema module:
 
 ```elixir
-defmodule MyProject.Message do
-  use MyProject.Web, :model
+defmodule MyProject.Messages.Message do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias MyProject.Messages.Message
```

Comment:
Please separate `use`, `import` and `alias` with empty lines.

Reply:
I agree with your style convention. But as per the phoenix guide, they style them with no empty lines. (I don't know why because the controllers are using empty lines on contrary.) This is which I'm confusing now.

Just scroll up some lines on this guide :
https://hexdocs.pm/phoenix/contexts.html#in-context-relationships

Reply:
This is not a phoenix part so we can separate these lines here.